scripts/config-diff.sh

```diff
@@ -232,12 +232,6 @@ function main {
 
     kayobe_init
 
-    # We need to use the same path for source and target to avoid noise in the diff output.
-    # Example: https://github.com/openstack/kolla-ansible/blob/5e638b757bdda9fbddf0fe0be5d76caa3419af74/ansible/roles/common/templates/td-agent.conf.j2#L9
```

Comment:
You would need to have custom fluentd config to trigger this issue.

Reply:
Okay. So a function to _normalise_ the paths within generated configs would be required.